Providers/Scripts/2.4x-2.5x/Scripts/nxOMSContainers.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env python
+#============================================================================
+# Copyright (C) Microsoft Corporation, All rights reserved.
+#============================================================================
+
+from contextlib import contextmanager
+
+import os
+import sys
+import imp
+import re
+import pwd
+import grp
+import codecs
+import shutil
+
+protocol = imp.load_source('protocol', '../protocol.py')
+nxDSCLog = imp.load_source('nxDSCLog', '../nxDSCLog.py')
+
+LG = nxDSCLog.DSCLog
+
+MODULE_RESOURCE_DIR ='/opt/microsoft/omsconfig/modules/nxOMSContainers/DSCResources/MSFT_nxOMSContainersResource/containers'
+
+PLUGIN_PATH = '/opt/microsoft/omsagent/plugin/'
+CONF_PATH = '/etc/opt/microsoft/omsagent/<WorkspaceId>/conf/omsagent.d'
```

Comment:
In AMS, will you only allow this resource to be applied to OMSAgents versions >=1.3? Otherwise, you need to support the case where the configuration path is just `/etc/opt/microsoft/omsagent/conf/omsagent.d`

Reply:
Agent version needs to be checked. Since with the OMI change its not compatible with the older version anymore. Need to check for newer agent.